support-frontend/assets/components/priceCards/priceCards.tsx

```diff
@@ -73,13 +74,25 @@ export function PriceCards({
 	onAmountChange,
 	paymentInterval,
 	otherAmountField,
+	hideChooseYourAmount,
 }: PriceCardsProps): JSX.Element {
-	const otherAmountLabel = amounts.length % 2 ? 'Other' : 'Choose your amount';
+	let currentAmountsLen = hideChooseYourAmount
```

Comment:
I got very confused by `currentAmountsLen` etc and ended up pulling down the branch to work it out!
I wonder if the intent (finding the width of the final button) could be clearer in this file, what do you think of something like this? https://github.com/guardian/support-frontend/pull/4706

Reply:
Merged